test/items.html

```diff
@@ -414,6 +422,97 @@
           expect(menuComponents()[0].getAttribute('aria-expanded')).to.equal('false');
         });
       });
+
+      describe('theme propagation', () => {
+        let subMenu2;
+
+        beforeEach(done => {
+          rootMenu = fixture('theme');
+          rootMenu.openOn = OPENEVENT;
+          target = rootMenu.firstElementChild;
+          rootMenu.items = [
+            {text: 'foo-0', children:
+              [
+                {text: 'foo-0-0'},
+                {text: 'foo-0-1', children: [
+                  {text: 'foo-0-1-0'}
+                ]},
+              ]
+            },
+            {text: 'foo-1'}
+          ];
+          open();
+          flush(() => {
+            open(menuComponents()[0]);
+            subMenu = getSubMenu();
+            flush(() => {
+              open(menuComponents(subMenu)[1]);
+              subMenu2 = getSubMenu();
+              flush(done);
+            });
+          });
+        });
+
+        function themeAttributeEquals(theme, el, overlay = false, listBox = false, items = false) {
```

Comment:
This helper is strange. It runs different checks depending on values of arbitrary boolean parameters that we then enable/disable differently for different tests...just test all the attributes in one go.

Reply:
Removed and tests are simplified. Now one test is testing all three `context-menus` and their nested elements.